lib/integrations/awsoidc/create_ec2ice_test.go

```diff
@@ -31,36 +32,68 @@ import (
 )
 
 type mockCreateEC2ICEClient struct {
-	name string
-	err  error
+	subnetToNameMap map[string]string
```

Comment:
nit: ditch the "Map" suffix?

```suggestion
	subnetToName map[string]string
```

I'd rather use a stronger type here too, but this is not as important in a fake as in the public APIs.